src/block/progress-circle/save.js

```diff
@@ -0,0 +1,62 @@
+import ProgressCircleStyles from './style'
+import {
+	BlockDiv,
+	CustomCSS,
+	getResponsiveClasses,
+} from '~stackable/block-components'
+import { version as VERSION } from 'stackable'
+import { withVersion } from '~stackable/higher-order'
+import classnames from 'classnames'
+
+import { InnerBlocks } from '@wordpress/block-editor'
+import { compose } from '@wordpress/compose'
+
+const Save = props => {
+	const { className, attributes } = props
+	const responsiveClass = getResponsiveClasses( attributes )
+
+	const blockClassNames = classnames( [
+		className,
+		'stk-block-progress-circle',
+		responsiveClass,
+	] )
+
+	const divClassNames = classnames( [
+		'stk-progress-circle',
+		{
+			'with-animation': attributes.progressAnimate,
```

Comment:
Let's prefix this into a modifier class:
```suggestion
			'stk--with-animation': attributes.progressAnimate,
```